ssl/ssl_locl.h

```diff
@@ -2263,7 +2265,7 @@ __owur int tls13_update_key(SSL *s, int send);
 __owur int tls13_hkdf_expand(SSL *s, const EVP_MD *md,
                              const unsigned char *secret,
                              const unsigned char *label, size_t labellen,
-                             const unsigned char *hash,
+                             const unsigned char *data, size_t datalen,
```

Comment:
Ah, *there*!  It escaped me through 3 readings...  thanks for pointing it out and sorry for the noise.